src/app/(frontend)/ticket/[ticketCode]/page.client.tsx

```
@@ -0,0 +1,158 @@
+    >
+      {/* Header */}
+      <header className="p-6 pb-4 border-b border-gray-100">
+        <div className="flex justify-between items-start">
+          <div>
+            <span className="inline-flex items-center bg-gray-200 text-gray-800 text-xs font-bold uppercase tracking-wider rounded-md px-3 py-1.5">
+              {t('ticket.ticket')}
+            </span>
+            {event?.title && (
+              <h2
+                className="text-2xl font-bold mt-2"
+                dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(event.title) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)